docs/src/components/App/Header/Header.js

```diff
@@ -1,18 +1,81 @@
 import styles from './Header.less';
 
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
-import { Link } from 'react-router-dom';
-import { PageBlock, Card, Section, Text, ScreenReaderOnly } from 'seek-style-guide/react';
+import { Link, withRouter } from 'react-router-dom';
+import {
+  PageBlock,
+  Card,
+  Highlight,
+  Section,
+  ChevronIcon,
+  Text,
+  TextField,
+  ScreenReaderOnly
+} from 'seek-style-guide/react';
 import Logo from './Logo/Logo';
 import demoSpecExports from '../../../../../react/*/*.demo.js';
 const demoSpecs = demoSpecExports.map(x => x.default);
+const allRoutes = [
+  { route: '/typography', title: 'Typography', category: 'Guides' },
+  { route: '/page-layout', title: 'Page Layout', category: 'Guides' },
+  { route: '/sandbox', title: 'Sandbox', category: 'Tools' },
+  ...demoSpecs
+];
 
-export default class Header extends Component {
+const SEARCH_BAR_ID = 'search-bar-field';
+
+const WithHighlighting = ({ highlighted, children }) => {
```

Comment:
Wonder if it would be worth moving some of these functions into another file. This one has become rather long

Reply:
I've split out a bit of the sorting logic into a new file, as well as the WithHighlighting component